distribution/docs/src/main/resources/_contents/_tables/org.codice.admin.ui.configuration-table-contents.adoc

```diff
@@ -0,0 +1,76 @@
+.[[org.codice.admin.ui.configuration]]Admin UI Configuration
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|Enable System Usage message
+|systemUsageEnabled
+|Boolean
+|Turns on a system usage message,which is shown when the Admin Application is opened
+|false
+|true
+
+|System Usage Message Title
+|systemUsageTitle
+|String
+|A title for the system usage message when the application is opened
+|
+|true
+
+|System Usage Message
+|systemUsageMessage
+|String
+|A system usage message to be displayed to the user each time the user opens the application
```

Comment:
I notice that below, where there is a two-sentence description, you're including a terminal period on each sentence. It appears that elsewhere, where descriptions are one-sentence long, you're eschewing the period. That's a little oddly inconsistent to me.